### PR TITLE
build: Use qrc file content as source

### PR DESCRIPTION
**Pull request by promag**
Use `rcc --list` instead of also adding files in the Makefile.

### CODE REVIEW

**Review by promag — COMMENTED** [review]
@hebasto @fanquake I have no idea if this is a good approach, but the goal is to avoid adding new files to Makefile, just add to the .qrc file.

**Comment by hebasto** [comment]
Rebase?

**Review by jarolrod — COMMENTED** [review]
Concept ACK

This approach/idea is better than manually keeping a list of our components and pages. Rebase

**Comment by promag** [comment]
I'll rebase and try to sharpen my lame makefile skillz to make it work.

File: src/Makefile.qt.include [hunk 1]
```diff
@@ -296,12 +296,7 @@ QML_RES_FONTS = \
 
 QML_QRC_CPP = qml/qrc_bitcoin.cpp
 QML_QRC = qml/bitcoin_qml.qrc
-QML_RES_QML = \
-  qml/components/BlockCounter.qml \
-  qml/components/ConnectionOptions.qml \
-  qml/controls/OptionButton.qml \
-  qml/pages/initerrormessage.qml \
-  qml/pages/stub.qml
+$(eval QML_RES_QML = $(shell $(RCC) --list $(QML_QRC)))
```
Comment (hebasto) [inline src/Makefile.qt.include:299]: Maybe
```suggestion
QML_RES_QML :=  $(shell $(RCC) --list $(QML_QRC))
```
?

Comment (hebasto) [reply src/Makefile.qt.include]: My suggestion does not work due to Automake [limitations](https://www.gnu.org/software/automake/manual/html_node/Conditional-Sources.html).

**Comment by promag** [comment]
Closing since workarounds/hacks are needed to address limitation `configure substitutions are not allowed in _SOURCES variables`.

**Pull request closed without merging**